fluent.conf

```diff
@@ -48,6 +48,12 @@
   tag php.app-access
 </source>
 
+<filter php.*>
+  @type map
+  time time
+  record ({"php_error_log" => {"error" => record["msg"].strip, "level" => record["severity"].upcase}})
```

Comment:
not sure you can do this here, as it requires ruby

Reply:
I tested in [v1.3.1](https://github.com/AnchorFree/php-logger/releases/tag/v1.3.1) and it worked you can see example result in this [comment](https://github.com/AnchorFree/php-logger/pull/8#issuecomment-373881403).

Reply:
@onorua I didn't get you comment at first but you were right. I removing `.strip` and `. upcase ` it for now. It will help with those warnings:
```
undefined method `upcase' for nil:NilClass
```